_api-reference/search-apis/search.md

```diff
@@ -30,51 +30,47 @@ All parameters are optional.
-seq_no_primary_term | Boolean | Whether to return sequence number and primary term of the last operation of each document hit.
-size | Integer | How many results to include in the response.
-sort | List | A comma-separated list of &lt;field&gt; : &lt;direction&gt; pairs to sort by.
-_source | String | Whether to include the `_source` field in the response.
-_source_excludes | List | A comma-separated list of source fields to exclude from the response.
-_source_includes | List | A comma-separated list of source fields to include in the response.
-stats | String | Value to associate with the request for additional logging.
-stored_fields | Boolean | Whether the get operation should retrieve fields stored in the index. Default is `false`.
-suggest_field | String | Fields OpenSearch can use to look for similar terms.
-suggest_mode | String | The mode to use when searching. Available options are `always` (use suggestions based on the provided terms), `popular` (use suggestions that have more occurrences), and `missing` (use suggestions for terms not in the index).
-suggest_size | Integer | How many suggestions to return.
-suggest_text | String | The source that suggestions should be based off of.
-terminate_after | Integer | The maximum number of matching documents (hits) OpenSearch should process before terminating the request. Default is 0.
-timeout | Time | How long the operation should wait for a response from active shards. Default is `1m`.
-track_scores | Boolean | Whether to return document scores. Default is `false`.
-track_total_hits | Boolean or Integer | Whether to return how many documents matched the query.
-typed_keys | Boolean | Whether returned aggregations and suggested terms should include their types in the response. Default is `true`.
-version | Boolean | Whether to include the document version as a match.
+allow_no_indices | Boolean | Whether to ignore wildcards that don’t match any indexes. Default is `true`. Example: `GET test-index-*/_search?allow_no_indices=true` |
+allow_partial_search_results | Boolean | Whether to return partial results if the request runs into an error or times out. Default is `true`. Example: `GET test-index/_search?allow_partial_search_results=false` |
+analyzer | String | The analyzer to use in the query string. Requires `q=` or a `query_string` body. Example: `GET test-index/_search?q=title:test&analyzer=standard` |
+analyze_wildcard | Boolean | Whether the update operation should include wildcard and prefix queries in the analysis. Default is `false`. Requires `q=`  or `query_string`. Example: `GET test-index/_search?q=title:te*&analyze_wildcard=true` |
+batched_reduce_size | Integer | How many shard results to reduce on a node. Default is `512`. Example: `GET test-index/_search?batched_reduce_size=2` |
```

Comment:
This refers to the coordinating node, correct? "To reduce" is a bit unclear. Maybe:
```suggestion
batched_reduce_size | Integer | The number of shard results to combine into one batch on the coordinating node before returning the final search results. Limits the number of shard results processed together, helping reduce memory usage when a search request spans many shards. Default is `512`. Example: `GET test-index/_search?batched_reduce_size=2`. |
```